plugins/rtmp-services/data/services.json

```diff
@@ -2864,6 +2864,17 @@
                 "max video bitrate": 6000,
                 "max audio bitrate": 160
             }
+        },        
+        {
+            "name": "StreamVi",
+            "common": true,
```

Comment:
Please don't set your service to common.

Reply:
Sorry, I fixed the commit.